init-psql-db.nix

```diff
@@ -0,0 +1,17 @@
+''
+export PGDATA=$PWD/postgres_data
+export PGHOST=$PWD/postgres
+export LOG_PATH=$PWD/postgres/LOG
+export PGDATABASE=postgres
+export DATABASE_URL="postgresql:///postgres?host=$PGHOST"
+if [ ! -d $PGHOST ]; then
+  mkdir -p $PGHOST
+fi
+if [ ! -d $PGDATA ]; then
+  echo 'Initializing postgresql database...'
+  initdb $PGDATA --auth=trust >/dev/null
+fi
+pg_ctl start -l $LOG_PATH -o "-c listen_addresses= -c unix_socket_directories=$PGHOST"
+
+sleep 0.1 # for template1 missing error
```

Comment:
although, in general, sleeps should not be needed, ever!? So that seems odd ...

Reply:
> maybe you want the sleep before the `pg_ctl start` not after? I'm not sure, but I think the `template1 missing` error omes from initdb, which is trying to make a copy of that template, when it creates a new db. !?

Tried it before `pg_ctl start` but it still fails.

> although, in general, sleeps should not be needed, ever!? So that seems odd ...

I agree. #15